deps/build.jl

```diff
@@ -4,10 +4,10 @@ module DefaultDeps
     if isfile("deps.jl")
         include("deps.jl")
     end
-    if !isdefined(:ROOTENV)
+    if !isdefined(@__MODULE__, :ROOTENV)
```

Comment:
Errr, I think you need Compat... The using above is not in the right module iiuc.